packages/perseus/src/widgets/radio/base-radio.jsx

```diff
@@ -555,6 +374,7 @@ class BaseRadio extends React.Component<$FlowFixMe, $FlowFixMe> {
                                     ) {
                                         this.updateChoice(i, {
                                             checked: !choice.checked,
+                                            crossedOut: choice.crossedOut,
```

Comment:
**question:** This feels a little risky.  Are there any situations where this would cause a change in behavior?

Reply:
This was because I was trying to get around my lack of Flow understanding. I double checked and I agree that it's risky than I originally thought, so I went through and fixed the Flow stuff.